pep/app/pods/application/controller.ts

```diff
@@ -43,6 +48,85 @@ export default class Application extends Controller {
     @tracked viewedPeriod: ViewPeriod = ViewPeriod.PAST_WEEK;
     @tracked searchTerms: SearchTermValue[] = [];
 
+    @tracked shortcuts: KeyboardShortcut[] = [
+        {
+            keys: NAVIGATE_TO_BROWSE,
+            shortcut: () => {
+                this.transitionToRoute('browse');
+            }
+        },
+        {
+            keys: NAVIGATE_TO_HOME,
+            shortcut: () => {
+                this.transitionToRoute('index');
+            }
+        },
+        {
+            keys: NAVIGATE_TO_SEARCH,
+            shortcut: () => {
+                this.transitionToRoute('search');
+            }
+        },
+        {
+            keys: CLEAR_SEARCH,
+            shortcut: () => {
+                this.clearSearch();
+            }
+        },
+        {
+            keys: TOGGLE_SIDEBARS,
+            shortcut: () => {
+                const areBothOpen = this.sidebar.leftSidebarIsOpen && this.sidebar.rightSidebarIsOpen;
+                this.sidebar.toggleAll(areBothOpen ? false : true);
+            }
+        },
+        {
+            keys: TOGGLE_LEFT_SIDEBAR,
+            shortcut: () => {
+                this.sidebar.toggleLeftSidebar(!this.sidebar.leftSidebarIsOpen);
+            }
+        },
+        {
+            keys: TOGGLE_RIGHT_SIDEBAR,
+            shortcut: () => {
+                this.sidebar.toggleRightSidebar(!this.sidebar.rightSidebarIsOpen);
+            }
+        },
+        {
+            keys: TOGGLE_GLOSSARY_HIGHLIGHTING,
+            shortcut: () => {
+                if (this.currentUser.preferences) {
+                    const currentValue = this.currentUser.preferences?.glossaryFormattingEnabled;
+                    this.currentUser.updatePrefs({ [PreferenceKey.GLOSSARY_FORMATTING_ENABLED]: !currentValue });
+                }
+            }
+        },
+        {
+            keys: TOGGLE_HIC,
+            shortcut: () => {
+                if (this.currentUser.preferences) {
+                    const currentValue = this.currentUser.preferences?.searchHICEnabled;
+                    this.currentUser.updatePrefs({ [PreferenceKey.SEARCH_HIC_ENABLED]: !currentValue });
+                }
+            }
+        },
+        {
+            keys: ESCAPE,
+            shortcut: () => {
+                const active = document.activeElement;
+                const enteringText =
+                    active instanceof HTMLElement &&
+                    (active.isContentEditable || active.tagName === 'INPUT' || active.tagName === 'TEXTAREA');
```

Comment:
this should include `SELECT` since entering text on a select dropdown is valid (it causes the first matching option to be selected), could also clean this up to: 
```ts
['INPUT', 'TEXTAREA', 'SELECT'].includes(active.tagName)
```